README.md

```diff
@@ -1,7 +1,24 @@
 # Backend deployment and database challenge
 
 ## My deployed API URL is:
-(Paste your render URL here)
+[https://users-app-assessment-backend-cl.onrender.com](https://users-app-assessment-backend-cl.onrender.com)
+
+## Endpoints:
+| Resource | Method | Route       | Description                         |
+|----------|--------|-------------|-------------------------------------|
+| app      | ALL    | *           | Error message                       |
+| app      | GET    | /           | Healthcheck route                   |
+| users    | GET    | /users      | Get all users                       |
+| users    | GET    | /users/{id} | Return one user by their id of {id} |
+
+## Questions:
+1. What is the difference between `float`, `double`, and `decimal` data types for latitude and longitude in SQL?
```

Comment:
Great question! I **think** that if you needed 100% perfect accuracy, you would want either `double` or `decimal` but not `float`. The `float` type can sometimes have small errors because computers (base 2) are not always able to correctly represent every kind of decimal number (in base 10). `double` and `decimal` use more storage in order to be more accurate.

For sure, whenever you are dealing with money you should use `double` (or `decimal`) and definitely not `float`.


More info here: https://www.postgresql.org/docs/current/datatype-numeric.html